javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentInstaller.java

```diff
@@ -540,5 +541,11 @@ private static boolean isJavaBefore9WithJfr() {
     return Thread.currentThread().getContextClassLoader().getResource(jfrClassResourceName) != null;
   }
 
+  private static void logVersionInfo() {
+    VersionLogger.logAllVersions();
```

Comment:
Can we add a test for this? I think we don't even need a smoke test anymore since we updated the normal ones to use the agent.

Reply:
Hmm, okay, but how do you test that? It's basically a log statement. Do you have any suggestion, anything in mind?

Reply:
If it's easier actually, then smoke test is fine too. We have smoke tests checking ID injection I think so probably right there should work.